src/index.js

```diff
@@ -63,6 +64,10 @@ function IpfsAPI (host_or_multiaddr, port) {
       opts = {}
     }
 
+    if (typeof files === 'string' && files.startsWith('http')) {
+      files = request(files)
```

Comment:
`23:09 <+daviddias> dignifiedquire: interesting, so our thing understands 'strings', 'paths', 'buffers' and 'streams' too?``

Sweet!